source/css/sass/mixins/_spacing.scss

```diff
@@ -91,7 +91,9 @@
           }
 
           html[dir] & {
-            #{$space-type}-inline: rem($firstValue $secondValue);
+            // Not using shorthand due to lack of browser support.
```

Comment:
Could always `@supports`. Does make me wonder if we should go down the PostCSS route and write SCSS in the format that we want rather than using mixins...

Reply:
(Mentioned in https://github.com/libero/libero/issues/88#issuecomment-459299971)